script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -359,44 +360,45 @@ Breathing as hotly and heavily as a ■■■■―――
 -- Page 209, Offset 4521.
 -- @g　白い<獲物|おんな>の姿を求める。
 @g
-I hunt the figure of the <woman|p r e y  > in white.
+I desire her. I desire the <woman|p r e y  > in white.
 }
 [sha:ce4cc096c8926934d1b1a3580f9437b4c1ca8989]{
 -- Page 210, Offset 4522.
 -- 「はあ――――はあ――――はあ――――」
-"%{i}Haa――haa――haa―――%{/i}"
+"%{i}Hahh... hahh... hahh...%{/i}"
 }
 [sha:7f7495da9ecd4a2bc39e6af3017dc4e1e57ced1a]{
 -- Page 210, Offset 4523.
 -- @g　女はゆっくりと歩いている。
 @g
-The woman is walking slowly.
+Her pace is casual.
 }
 [sha:e8fceaf7ac6b711e3f83095d81c16306efec99ad]{
 -- Page 210, Offset 4524.
 -- @g　オレにはいっさい関心がない。
 @g
-She's not interested in me at all.
+She's not paying me the slightest attention.
 }
 [sha:3ffab80274d9ad56f06d8f44d372935ecf4ae6e0]{
 -- Page 210, Offset 4525.
 -- @g　―――かすかな疑問が脳裏をかすめる。
 @g
-―――A faint question crosses my mind.
+A faint question crosses my mind.
 }
 [sha:25855c89f2006dd9b8d20a7f567628288922f577]{
 -- Page 210, Offset 4526.
 -- @g　関心がない？　そんな馬鹿な。アレがそんなタマか。だが事実として気がついてさえいない。なにかの後か。なにかの後だろう。おそらくひどく疲れている。消耗している。出し切っている。だからこそ機会がある。オレがこんな気持ちになるのは、つまるところは／■しきれると／ 確信があるからだ。
 // タマ here - (11) female entertainer (e.g. a geisha); (12) (derog) person (when commenting on their nature); ?
 @g
-Not interested? That's absurd.
- Is she really that sort of person?
- But the truth is, she hasn't even noticed me.
+Really? She's not paying me any attention?
+ How is that possible?
+ She's not one of %{i}those%{/i} people, is she?
```

Comment:
These italics don't do anything because the whole sentence is already italicized by the `@g` marker

Reply:
Guess It'll just have to go?